### DISCUSSION TITLE
Error tidak bisa buka link video capstone project

### DISCUSSION THREAD

**Discussion by npmitaart** [teamalgoritma/community#40] [Q&A] [answered]
Halo tim teaching assistants, saya mau bertanya perihal link video dari capstone project saat mau diakses tidak bisa dibuka, kira-kira solusinya bagaimana ya? Terima kasih :D
![Screenshot (788)](https://user-images.githubusercontent.com/50654221/162355235-99fd8ba0-2177-4d6c-a356-011ce791de68.png)

**Comment by inytss** [comment, marked as answer]
Halo @npmitaart terima kasih telah menginformasikan hal ini. Untuk link Video Capstone DA dapat Bapak akses melaui [link berikut](https://drive.google.com/drive/folders/19HTlrk0elqgaDtxLoVTExZ3m-Q8eytaL?usp=sharing). Atau Bapak dapat mengaksesnya pada pandual class guide yang sudah kami update. Semoga membantu ya Pak :)
